lib/Serialization/ModuleFormat.h

```diff
@@ -1257,7 +1257,6 @@ namespace decls_block {
     BCFixed<1>,             // implicit flag
     BCFixed<1>,             // class-bounded?
     BCFixed<1>,             // objc?
-    BCFixed<1>,             // existential-type-supported?
```

Comment:
Don't forget to bump the format version.

Reply:
Oops. Does it still need to be (has been incremented 6 times since this landed)?